src/container/Search/Search.tsx

```diff
@@ -43,29 +39,31 @@ const Search = () => {
 
     setLoading(true)
 
-    const blocks = axios.post(getApiUrl(ApiUrls.SEARCH_BLOCKS), {
-      network_identifier: networkIdentifier,
-      limit: 8,
-      query: value.toUpperCase(),
+    const blockSearchParams = new URLSearchParams({
+      search: value,
+      with_transactions: 'true',
     })
-
-    const transactions = axios.post(getApiUrl(ApiUrls.SEARCH_TRANSACTIONS), {
-      network_identifier: networkIdentifier,
-      limit: 5,
-      transaction_identifier: { hash: value.toUpperCase() },
+    const transactionSearchParams = new URLSearchParams({
+      search: value,
+      with_blocks: 'true',
     })
 
+    const blocks = axios.get(getApiUrl(ApiUrls.SEARCH_BLOCKS) + blockSearchParams.toString())
+    const transactions = axios.get(
+      getApiUrl(ApiUrls.SEARCH_TRANSACTIONS) + transactionSearchParams.toString(),
+    )
+
     Promise.all([transactions, blocks]).then((values) => {
       let transactions: Transaction[] = []
       let blocks: Block[] = []
 
       for (let i = 0; i < values.length; i++) {
-        const { data } = values[i]
-        if (data && data.transactions) {
-          transactions = formatSearchTransactionsFromJson(data)
+        const { data } = values[i].data
+        if (data[0] && data[0].object === 'transaction') {
```

Comment:
Minor, but if you're writing defensively you can also be declarative and DRY in your assertions:
```suggestion
        const first = data[0]
        if (first) {
          if (first.object === 'transaction') {
```
Since this is TS, I believe there's a more elegant elvis-operator style dealie but I always get it wrong `a?.b?.c?` so I won't attempt it here.